compiler/mx.compiler/suite.py

```diff
@@ -4,8 +4,8 @@
   "sourceinprojectwhitelist" : [],
 
   "groupId" : "org.graalvm.compiler",
-  "version" : "24.2.1.0",
-  "release" : True,
+  "version" : "24.2.1.1",
```

Comment:
I know this is the script doing it, but we should probably change this to `24.2.2.0` over `24.2.1.1`. The July release will likely be `24.2.2.0-Final`.

Reply:
Right, we either bump straight to 24.2.2.0 and patch the version only if required to make a pre-CPU release or bump it to 24.2.1.1 now and bump it later to 24.2.2.0 (when merging upstream changes). The latter is what we have been doing so far. I am OK with either approach.

Reply:
I have no objections to changing the scheme as long as it keeps being automated in: https://github.com/graalvm/mandrel-packaging/blob/master/release-scripts/mandrel-release.java

Perhaps could be wrapped up in the change: https://github.com/graalvm/mandrel-packaging/issues/490 because not having this automated essentially makes the JDK 21 suite.py changes a manual effort...